clinical-mdr-api/consumer_api/requirements/generate_traceability.py

```
@@ -60,73 +60,103 @@ def extract_section_headers_and_content(
+                                        new_th = soup.new_tag("th")
+                                        new_th.string = "Test Status"
+                                        header_row.append(new_th)
+
+                                rows = table_sibling.find_all("tr")
+                                for row in rows[1:]:
+                                    cols = row.find_all("td")
+                                    if len(cols) >= 2:
+                                        test_file = cols[0].get_text().strip()
+                                        test_method = cols[1].get_text().strip()
+                                        tests_list.append({"file": test_file, "method": test_method})
+                                        
```

Pylint (4.0.5)

[convention] Trailing whitespace

(C0303)